package.json

```diff
@@ -13,16 +13,17 @@
     "cueball": "2.7.1",
     "digest-stream": "0.2.2",
     "forkexec": "1.1.0",
+    "fs-ext": "1.3.0",
     "imgmanifest": "3.1.0",
     "jsprim": "2.0.0",
     "kstat": "1.0.1",
     "lazy-property": "1.0.0",
-    "lockfd": "1.2.0",
     "lstream": "0.0.4",
     "mkdirp": "^0.3.4",
     "nodeunit": "0.9.1",
     "once": "1.3.0",
     "pty.js": "0.3.1",
+    "qlocker": "git+https://github.com/joyent/node-qlocker.git#ec3c5323e4070fa83b39508f7199efab4e6d055b",
```

Comment:
Nit - can qlocker be published in npm - as mostly we prefer npm over git urls.

BTW - curious on why we don't use an existing node lockfile module, e.g.
https://www.npmjs.com/package/lockfile

Reply:
> Nit

ok, published

> BTW

That only handles inter-process locks.  We need inter-process and intra-process locks.

```
$ cat test
#! /usr/bin/env node

lf = require('lockfile')

lf.lock('/tmp/lock', function() {
    lf.lock('/tmp/lock', function() {
        console.log('got it twice')
    })
})

$ ./test
got it twice
```

Reply:
Thanks - that helps :)